doc/source/whatsnew/v0.18.2.txt

```diff
@@ -528,3 +528,5 @@ Bug Fixes
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
 - Bug in ``groupby`` with ``as_index=False`` returns all NaN's when grouping on multiple columns including a categorical one (:issue:`13204`)
+
```

Comment:
thanks for the tip, added blank lines before my bugfix entry. hope that helps with the merge.